src/js/hca-rjsf/config/form.js

```diff
@@ -128,7 +132,9 @@ const formConfig = {
         veteranInformation: {
           path: 'veteran-information/personal-information',
           title: 'Veteran information',
-          initialData: {},
+          initialData: {
+            maritalStatus: 'Married'
```

Comment:
Should this (and the other `initialData`s) still be here? Are you just setting defaults?